packages/ember-htmlbars/tests/integration/component_invocation_test.js

```diff
@@ -53,516 +34,6 @@ QUnit.module('component - invocation', {
-  var SampleComponent = Component.extend();
-  SampleComponent.reopenClass({
-    positionalParams: ['name', 'age']
-  });
-  owner.register('template:components/sample-component', compile('{{attrs.name}}{{attrs.age}}'));
-  owner.register('component:sample-component', SampleComponent);
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    layout: compile('{{sample-component "Quint" 4}}')
-  }).create();
-
-  runAppend(view);
-
-  equal(jQuery('#qunit-fixture').text(), 'Quint4');
-});
-
-QUnit.test('dynamic named positional parameters', function() {
-  var SampleComponent = Component.extend();
-  SampleComponent.reopenClass({
-    positionalParams: ['name', 'age']
-  });
-
-  owner.register('template:components/sample-component', compile('{{attrs.name}}{{attrs.age}}'));
-  owner.register('component:sample-component', SampleComponent);
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    layout: compile('{{sample-component myName myAge}}'),
-    context: {
-      myName: 'Quint',
-      myAge: 4
-    }
-  }).create();
-
-  runAppend(view);
-
-  equal(jQuery('#qunit-fixture').text(), 'Quint4');
-  run(function() {
-    set(view.context, 'myName', 'Edward');
-    set(view.context, 'myAge', '5');
-  });
-
-  equal(jQuery('#qunit-fixture').text(), 'Edward5');
-});
-
-QUnit.test('if a value is passed as a non-positional parameter, it takes precedence over the named one', function() {
-  var SampleComponent = Component.extend();
-  SampleComponent.reopenClass({
-    positionalParams: ['name']
-  });
-
-  owner.register('template:components/sample-component', compile('{{attrs.name}}'));
-  owner.register('component:sample-component', SampleComponent);
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    layout: compile('{{sample-component notMyName name=myName}}'),
-    context: {
-      myName: 'Quint',
-      notMyName: 'Sergio'
-    }
-  }).create();
-
-  expectAssertion(function() {
-    runAppend(view);
-  }, `You cannot specify both a positional param (at position 0) and the hash argument \`name\`.`);
-});
-
-QUnit.test('static arbitrary number of positional parameters', function() {
-  var SampleComponent = Component.extend();
-  SampleComponent.reopenClass({
-    positionalParams: 'names'
-  });
-
-  owner.register('template:components/sample-component', compile('{{#each attrs.names as |name|}}{{name}}{{/each}}'));
-  owner.register('component:sample-component', SampleComponent);
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    layout: compile('{{sample-component "Foo" 4 "Bar" id="args-3"}}{{sample-component "Foo" 4 "Bar" 5 "Baz" id="args-5"}}{{component "sample-component" "Foo" 4 "Bar" 5 "Baz" id="helper"}}')
-  }).create();
-
-  runAppend(view);
-
-  equal(view.$('#args-3').text(), 'Foo4Bar');
-  equal(view.$('#args-5').text(), 'Foo4Bar5Baz');
-  equal(view.$('#helper').text(), 'Foo4Bar5Baz');
-});
-
-QUnit.test('arbitrary positional parameter conflict with hash parameter is reported', function() {
```

Comment:
:heavy_check_mark: arbitrary positional parameter conflict with hash parameter is reported